### ISSUE TITLE
the appendQueryString function Having defects

### ISSUE THREAD

**Issue by feng524822** [krampstudio/aja.js#19]
sometimes, use aja().url('xxx.com').data({a: 1}).go();the request url will be 'xxx.com?&a=1', the query string start with ?&,it is not too bright, fix it with:
~~~javascript
var appendQueryString = function appendQueryString(url, params) {
        var key;
        url = url || '';
        if (params) {
            var querystring = '';
            if (typeof params === 'string') {
                querystring = params;
            } else if (typeof params === 'object') {
                for (key in params) {
                    querystring += '&' + encodeURIComponent(key) + '=' + encodeURIComponent(params[key]);
                }
            }

            if (querystring.indexOf('&') === 0) {
                querystring = querystring.substr(1);
            }

            url += (url.indexOf('?') === -1 ? '?' : '&');
            url += querystring;
        }

        return url;
    };
~~~

**Comment by krampstudio** [comment]
I'll have a look

**Comment by krampstudio** [comment]
merged but is going to be released as part of the 0.5.0 release
